changelog/unreleased/enhancement-redesign-status-indicators

```diff
@@ -0,0 +1,7 @@
+Enhancement: Redesign OcStatusIndicators
```

Comment:
Upon further consideration, I'd suggest adding two changelog items here - one is a `Change`, removing the sharing indicators from the OcResource component, and one is an `Enhancement`, redesigning the OcStatusIndicators - we could also discuss whether we want to introduce a `hasStatusIndicators` prop to the OcResource @kulmann

Reply:
Let's keep it simple and go with the two changelog items, i.e. your first proposal. We don't need to maintain the status indicators within the OcResource component.

Reply:
ok